docs/basics/ActionsManagers.md

```diff
@@ -0,0 +1,194 @@
+# Actions Managers
+
+>##### Not familiar with Observables/RxJS?
+> redux-observable requires an understanding of Observables with RxJS. If you're new to Reactive Programming with RxJS, head over to [http://reactivex.io/rxjs/](http://reactivex.io/rxjs/) to familiarize yourself first.
+
+An **Actions Manager** (or Manager for short) is the core primitive of redux-observable.
+
+It is a function which takes a stream of actions and returns a stream of actions. **Actions in, actions out.**
+
+You can think of it of having roughly this type signature:
+
+```js
+function (action$: Observable<Action>, store: Store): Observable<Action>;
+```
+
+While you'll most commonly produce actions out in response to some action you received in, keep in mind that's not actually a requirement! Once you're inside your Manager, use any Observable patterns you desire as long as anything output from the outermost stream is an action.
+
+The actions you emit will be immediately dispatched through the normal `store.dispatch()`, so under the hood redux-observable effectively does `manager(actions$, store).subscribe(store.dispatch)`
+
+> The pattern of handling side effects this way is similar to the *process manager* pattern, [often](http://kellabyte.com/2012/05/30/clarifying-the-saga-pattern/) [incorrectly](https://msdn.microsoft.com/en-us/library/jj591569.aspx) described as a *saga*. If you're familiar with [redux-saga](http://yelouafi.github.io/redux-saga/), redux-observable is very similar. But because it uses RxJS it is much more declarative and you utilize and expand your existing RxJS abilities.
+
+## A Basic Example
+
+Let's start with a simple Actions Manager example:
+
+```js
+const pingManager = action$ =>
+  action$.filter(action => action.type === 'PING')
+    .mapTo({ type: 'PONG' });
+    
+// later...
+dispatch({ type: 'PING' });
+```
+
+> Noticed how `action$` has a dollar sign at the end? It's simply a common RxJS convention to identify variables that reference a stream.
+ 
+`pingManager` will listen for actions of type `PING` and map them to a new action, `PONG`. This example is functionally equivalent to doing this:
+
+```js
+dispatch({ type: 'PING' });
+dispatch({ type: 'PONG' });
+```
+
+> Actions Managers run alongside the normal Redux dispatch channel, so you cannot "swallow" an action. When you map one action to another, you're not preventing the original action from reaching the reducers.
```

Comment:
This needs to be more clear that it's referring to incoming actions, not outgoing ones.